css/syntaxes.json

```diff
@@ -269,6 +302,12 @@
   "hue-rotate()": {
     "syntax": "hue-rotate( <angle> )"
   },
+  "id-selector": {
+    "syntax": "<hash-token>"
+  },
+  "ident-token": {
```

Comment:
`ident-token` is a token type and out of scope of this dict.
Btw, it has another syntax